.github/workflows/integration-test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Get Opendistro version
         id: opendistro_version
         run: |
-          echo "::set-output name=opendistro_version::$(jq -r '.version' ./kibana/plugins/opendistro_security/package.json)"
+          echo "::set-output name=opendistro_version::$(jq -r '.version' ./kibana/plugins/opendistro_security/package.json | sed 's/[0-9]*$/0/')"
```

Comment:
Just a question: will it always be this case or just for this release?

Reply:
It fails because kibana plugin is on 1.13.0.1 and es plugin is on 1.13.0.0. I found the release number here:
https://opendistro.github.io/for-elasticsearch-docs/docs/install/plugins/#security

The command is not that smart to retrieve this information but with build number 0 should be more common case.